tensorflow/compiler/xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -3370,6 +3370,319 @@ ENTRY test {
 ; CHECK:           backend_config="{
 ; CHECK-DAG:         \"alpha_real\":1
 ; CHECK-DAG:         \"alpha_imag\":0
+; CHECK-DAG:         "beta\":0
+; CHECK-DAG:         \"dot_dimension_numbers\":{
+; CHECK-DAG:           \"lhs_contracting_dimensions\":[\"1\"]
+; CHECK-DAG:           \"rhs_contracting_dimensions\":[\"0\"]
+; CHECK-DAG:           \"lhs_batch_dimensions\":[]
+; CHECK-DAG:           \"rhs_batch_dimensions\":[]
+; CHECK-DAG:         }
+; CHECK-DAG:         \"precision_config\":{
+; CHECK-DAG:           \"operand_precision\":[\"DEFAULT\",\"DEFAULT\"]
+; CHECK-DAG:         }
+; CHECK-DAG:         \"epilogue\":\"BIAS_RELU\"
+; CHECK:           }"
+      )");
+}
+
```

Comment:
Please also adjust existing tests which are failing after your change.

Reply:
I don't see how this is resolved. The pre-existing failing tests I noticed are:

LegacyCublasGemmRewriteTest.BF16GemmWithBias
CublasLtGemmRewriteTest.BF16GemmWithBias
CublasLtGemmRewriteTest.BatchedSharedVectorBias
CublasLtGemmRewriteTest.VectorBiasBF16Unpadded
CublasTestsBothLegacyAndLt/ParameterizedGemmRewriteTest.ComplexAlphaSimpleRewrite/0
CublasTestsBothLegacyAndLt/ParameterizedGemmRewriteTest.ComplexAlphaSimpleRewrite/1

Don't those tests fail for you as well? Maybe it is the same issue that the HLO ordering is not deterministic, and some CHECK-DAG needs to be used?

Reply:
All tests pass for me on A100 and V100 systems. The BatchedSharedVectorBias and ComplexAlphaSimpleRewrite tests shouldn't be affected by this change. Can you provide details on the failures?

Reply:
Ok, if all tests are passing for you, I guess it will be easier if I adjust the tests while merging.

Reply:
I can't say that it is clear to me why these tests are failing on your side. Some of them seem to be orthogonal to this change.